package.json

```diff
@@ -27,10 +27,11 @@
     "@mapbox/sphericalmercator": "^1.1.0",
     "@mapbox/tilebelt": "^1.0.1",
     "@mapbox/vector-tile": "^1.3.1",
+    "@mapbox/vt2geojson": "^1.1.5",
```

Comment:
@artemp why is `vt2geojson` added? Was it a missing dependency?

Reply:
It turned out I had a `test/clipping.test.js` which is not part of a source tree. I'll remove `@mapbox/vt2geojson`, thanks.

Reply:
Fixed in https://github.com/mapbox/vtcomposite/pull/108/commits/947c1c6f62cc2e84f7f655cb9592053cc5de8229